api/graphql/schema.graphql

```diff
@@ -878,17 +879,16 @@ type Query {
   # countApplicants returns the number of candidates matching its filters, and requires no special permissions.
   countApplicants(where: ApplicantFilterInput): Int! @count(model: "User")
   pool(id: UUID! @eq): Pool @find @can(ability: "view", query: true)
-  pools: [Pool]! @all @can(ability: "viewAny", model: "Pool")
```

Comment:
Very strange - there was a duplicate query in the schema.  It seems like the last definition in the file won so I removed this one.